tests/event_context/test_udf_event_context.py

```diff
@@ -16,36 +16,45 @@ def udf_wrapper():
     from exasol_advanced_analytics_framework.event_context.udf_event_context \
         import UDFEventContext
     from collections import OrderedDict
+    import json
 
     def run(ctx: UDFContext):
         wrapper = UDFEventContext(
             ctx, exa, OrderedDict([("t1", "a"), ("t3", "b"), ("t2", "c")]))
         df = wrapper.fetch_as_dataframe(10)
-        assert wrapper.column_names() == ["t1", "t2", "t3"]
-        assert wrapper.rowcount() == 100
+        return_str = json.dumps(str({
```

Comment:
The longer , I think about this test, the more I feel, we lag a something in the UDF mock, maybe we should provide a context implementation without the need to use the executor. I think, I will add a ticket for that.